concrete-core/docs/user/introduction.md

```diff
@@ -0,0 +1,72 @@
+# Introduction
+
+Welcome to the `concrete-core` guide !
```

Comment:
In english, I think there is no space before "!"

Reply:
This comment applies many times in the files, but I post it just once to avoid being too repetitive.